.node-version

```diff
@@ -1 +1 @@
-12.14.1
\ No newline at end of file
+18.12.0
```

Comment:
16.18.0 is latest (as of 2022-11-03) in the 16.x series, wouldn't that make more sense given the target of `node16`?

Reply:
Good spot! Thank you! Changed it to `16.18.1` which is the newest stable version. Tested it and seems to be working fine. It's a pity that the project seems kind of abandoned. 😢